app/src/main/java/org/systers/mentorship/view/adapters/TasksAdapter.kt

```diff
@@ -52,18 +53,19 @@ class TasksAdapter(
         }
         else{
             itemView.cbTask.setOnClickListener {
-                val builder = AlertDialog.Builder(context)
-                builder.setTitle(context.getString(R.string.mark_task_title))
-                builder.setMessage(context.getString(R.string.mark_task_message))
-                builder.setPositiveButton(context.getString(R.string.yes)){dialog, which ->
-                    itemView.cbTask.isChecked=true
-                    markTask(item.id)
-                }
-                builder.setNegativeButton(context.getString(R.string.no)){dialog,which ->
-                    itemView.cbTask.isChecked=false
-                }
-                val dialog: AlertDialog = builder.create()
-                dialog.show()
+//                val builder = AlertDialog.Builder(context)
+//                builder.setTitle(context.getString(R.string.mark_task_title))
+//                builder.setMessage(context.getString(R.string.mark_task_message))
+//                builder.setPositiveButton(context.getString(R.string.yes)){dialog, which ->
+//                    itemView.cbTask.isChecked=true
```

Comment:
Why is this commented?

Reply:
Commented this, because, here I am navigating to the Comments Fragment. I am not marking the tasks as completed or not here. That was what I was asking, whether we will be updating the tasks here itself, or in the comment fragment

Reply:
Here would be fine